packages/vector_graphics/lib/src/vector_graphics.dart

```diff
@@ -61,6 +61,7 @@ VectorGraphic createCompatVectorGraphic({
   String? semanticsLabel,
   bool excludeFromSemantics = false,
   Clip clipBehavior = Clip.hardEdge,
+  Duration transitionDuration = const Duration(milliseconds: 500),
```

Comment:
This doesn't just allow the transition, it forces it on for all apps. should this be opt in if a transition duration is specififed?